### PR TITLE
Fix issue if http-authentication is network activated

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *The upgrade process fails if http-authentication is network activated*

## Solution
- *Desactive the plugin even if it can be network activated.*

## PR Status
Work finished. Package_check, basic tests and upgrade from last version OK.  
Could be reviewed and tested.

## Validation
---
*Minor decision*
- [x] **Upgrade previous version** : JimboJoe
- [x] **Code review** : JimboJoe
- [x] **Approval (LGTM)** : JimboJoe
- [ ] **Approval (LGTM)** : 
- [x] **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20fix_upgrade%20(Official)/)  
When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.

### CODE REVIEW

**Review by JimboJoe — APPROVED** [review]
LGTM, code review and upgrade from previous version OK 👍

**Comment by maniackcrudelis** [comment]
Because the upgrade process can fail and remove the app (https://github.com/YunoHost/yunohost/pull/422), it would be better to not wait for 3 days before merging this PR in testing.
But only wait for the CI.

**Comment by JimboJoe** [comment]
Agreed!

**Comment by maniackcrudelis** [comment]
CI OK
Let's merge it.

**Review by frju365 — COMMENTED** [review]
it was ok for me too ;) late LGTM

**Pull request merged**
